worker/tests/test_app.py

```diff
@@ -55,35 +54,13 @@ async def test_ingest_file(
         await br.publish(file, list=env.ingest_queue_name)
         for i in range(5):
             await asyncio.sleep(1)
+            storage_handler.refresh()
             file_status = storage_handler.get_file_status(file.uuid, file.creator_user_uuid)
             if file_status.processing_status == status:
                 break
         else:
             raise Exception(f"File never went to expected status. Final Status {file_status.processing_status}")
 
         if expected_chunks:
-            chunks = list(
-                scan(
-                    client=es_client,
-                    index=f"{env.elastic_root_index}-chunk",
-                    query={
-                        "query": {
-                            "bool": {
-                                "must": [
-                                    {
-                                        "term": {
-                                            "metadata.parent_file_uuid.keyword": str(file.uuid),
-                                        }
-                                    },
-                                    {
-                                        "term": {
-                                            "metadata.creator_user_uuid.keyword": str(file.creator_user_uuid),
-                                        }
-                                    },
-                                ]
-                            }
-                        }
-                    },
-                )
-            )
+            chunks = storage_handler.get_file_chunks(file.uuid, file.creator_user_uuid)
```

Comment:
I'm hoping to remove all chunks functions from the storage_handler then remove those functions, the API functions and the Chunk class

Reply:
accepted